src/core_modules/capture-core/components/FeedbackBar/FeedbackBar.component.js

```diff
@@ -128,5 +128,6 @@ class FeedbackBar extends React.Component<Props> {
         );
     }
 }
+Index.displayName = 'FeedbackBar';
 
-export default withStyles(styles)(FeedbackBar);
+export const FeedbackBar = withStyles(styles)(Index);
```

Comment:
I know we discussed this before but could we maybe export this as `FeedbackBarComponent` to avoid having to rename it in the `container` file? Looks to me that this way we gonna have less name changing down the road.